routing/missioncontrol.go

```diff
@@ -919,10 +924,17 @@ func decodePaymentFailure(r io.Reader, val interface{}, _ *[8]byte,
 	return tlv.NewTypeForDecodingErr(val, "routing.paymentFailure", l, l)
 }
 
-// paymentFailureInfo holds additional information about a payment failure.
+// paymentFailure holds additional information about a payment failure.
 type paymentFailureInfo struct {
```

Comment:
nit: looks like this comment was renamed by accident and doesn't match anymore.

Reply:
right, that was a rebase error, great eye for details :)